posts/2021-01-22-ee9-messaging-security-21002-beta.adoc

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: "Jakarta EE 9 Messaging and Security packages now available with Open Liberty 21.0.0.2-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/austin0
+author_github: https://github.com/austin0
+seo-title: Jakarta EE 9 Messaging and Security packages now available with Open Liberty 21.0.0.2-beta - OpenLiberty.io
+seo-description: Three new Jakarta EE 9 packages, Jakarta Messaging 3.0, Jakarta Security 2.0, and Jakarta Batch 2.0, join the large stable of Jakarta EE 9 packages already available within Open Liberty.
+blog_description: Three new Jakarta EE 9 packages, Jakarta Messaging 3.0, Jakarta Security 2.0, and Jakarta Batch 2.0, join the large stable of Jakarta EE 9 packages already available within Open Liberty.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 9 Messaging and Security packages now available with Open Liberty 21.0.0.2-beta
+Austin Bailey <https://github.com/austin0>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+New in Open Liberty 21.0.0.2-beta are three new Jakarta EE 9 packages, Jakarta Messaging 3.0, Jakarta Security 2.0, and Jakarta Batch 2.0, that join the large stable of Jakarta EE 9 packages already available within Open Liberty. Open Liberty 21.0.0.2-beta also includes previews for 7 of the updated MicroProfile modules, including Config, Metrics, Fault Tolerance, Health, OpenAPI, Rest Client, and JWT. 
```

Comment:
~for 7 of the~
for 8 of the

Reply:
Fixed.

Reply:
Not sure if this got undone.  It says 7 again.  The list should have the 8th one though as well.  The missing one is OpenTracing.